Sources/ClientRuntime/Networking/Http/URLSession/FoundationStreamBridge.swift

```diff
@@ -73,25 +73,9 @@ class FoundationStreamBridge: NSObject, StreamDelegate {
     /// Actor used to enforce the order of multiple concurrent stream writes.
     private let writeCoordinator = WriteCoordinator()
 
-    /// A shared serial DispatchQueue to run the `perform`-on-thread operations.
-    /// Performing thread operations on an async queue allows Swift concurrency tasks to not block.
-    private static let queue = DispatchQueue(label: "AWSFoundationStreamBridge")
-
-    /// Foundation Streams require a run loop on which to post callbacks for their delegates.
-    /// All stream operations should be performed on the same thread as the delegate callbacks.
-    /// A single shared `Thread` is started and is used to host the RunLoop for all Foundation Stream callbacks.
-    private static let thread: Thread = {
-        let thread = Thread {
-            autoreleasepool {
-                let timer = Timer(timeInterval: TimeInterval.greatestFiniteMagnitude, repeats: true, block: { _ in })
-                RunLoop.current.add(timer, forMode: .default)
-                RunLoop.current.run(until: Date.distantFuture)
-            }
-        }
-        thread.name = "AWSFoundationStreamBridge"
-        thread.start()
-        return thread
-    }()
+    /// A shared serial DispatchQueue to run the stream operations.
+    /// Performing operations on an async queue allows Swift concurrency tasks to not block.
+    private let queue = DispatchQueue(label: "AWSFoundationStreamBridge")
```

Comment:
This queue is now a member of a single `FoundationStreamBridge` instead of a shared, static queue.

Reply:
The static thread is deleted because stream callbacks will now be delivered to `queue`, and the thread is unused.